src/app/shared/components/feature-card/feature-card.component.spec.ts

```diff
@@ -1,38 +1,53 @@
-import {async, ComponentFixture, TestBed} from '@angular/core/testing';
-
 import {FeatureCardComponent} from './feature-card.component';
-import {CommonModule} from '@angular/common';
+import {Component} from '@angular/core';
+import {Feature} from '../../models/Feature';
+import {initContext, TestContext} from '../../../../testing/test.context';
 import {MatButtonModule, MatCardModule, MatIconModule, MatSlideToggleModule} from '@angular/material';
-import {CUSTOM_ELEMENTS_SCHEMA} from '@angular/core';
 
-describe('FeatureCardComponent', () => {
-  let component: FeatureCardComponent;
-  let fixture: ComponentFixture<FeatureCardComponent>;
+const mockFeature1: Feature = {
+    uid: 'Feature_UID1',
+    description: `Feature Card1 Description`,
+    group: 'Group Name1',
+    enable: false,
+    permissions: ['ROLE_ADMIN', 'ROLE_USER', 'ROLE_ADMIN', 'ROLE_USER']
+};
 
-  beforeEach(async(() => {
-    TestBed.configureTestingModule({
-      imports: [
-        CommonModule,
+const mockFeature2: Feature = {
```

Comment:
why is mockFeature2 required ?

Reply:
This is to test the **on change should apply** at Line #48. Please let me know if you feel it is redundant.